src/api/Email/emailApi.tsx

```diff
@@ -0,0 +1,31 @@
+import { z } from "zod";
+import { spWebContext } from "src/api/SPWebContext";
+import { useMutation } from "@tanstack/react-query";
+
+const spEmail = z.object({
+  pcolId: z.string(),
+  To: z.array(z.string()),
+  CC: z.array(z.string()).optional(),
+  BCC: z.array(z.string()).optional(),
+  Subject: z.string(),
+  Body: z.string(),
+});
+
+type spEmail = z.infer<typeof spEmail>;
+
+export const useSendEmail = () => {
+  return useMutation({
+    mutationFn: async (pcolEmail: spEmail) => {
+      const logEmail = {
+        Title: pcolEmail.pcolId.toString(),
```

Comment:
I like the idea of logging which POCL it came from, however, this alone doesn't give us enough detail to quickly fiter, as it doesn't tell us which Program it came from.